### PR TITLE
Updated the tox and travis configs -- BOM-1553

### PR DESCRIPTION
**Pull request by mraarif**
Relevant JIRA issue [here](https://openedx.atlassian.net/browse/BOM-1553).

### CODE REVIEW

**Comment by awais786** [comment]
Add this constraint.
```
# Stay on an LTS release
django<2.3
```

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +1,31 @@
...
-      env: TOXENV=quality
-    - python: 3.5
-      env: TOXENV=docs
-
+  - python: 3.5
+    env: TOXENV=quality
+  - python: 3.5
+    env: TOXENV=docs
+  - python: 3.8
+    env: TOXENV=quality
```
Comment (morenol) [inline .travis.yml:16]: We can remove this from matrix and instread add a new env with TOXENV=quality which should be equivalent to have these two workers for quality test here.

File: .travis.yml [hunk 2]
```diff
@@ -1,42 +1,31 @@
...
+  - python: 3.8
+    env: TOXENV=docs
```
Comment (morenol) [inline .travis.yml:18]: Same as above

File: .travis.yml (hunk 1 above)
Comment (mraarif) [reply .travis.yml:16]: that's actually a very good suggestion, thanks

File: tox.ini [hunk 3]
```diff
@@ -1,36 +1,31 @@
 [tox]
-envlist = py35-django{111,20,21,22}-drf{39,latest},quality, docs
+envlist = py{35,38}-django{22,30}-drf{39,latest},quality, docs
```
Comment (morenol) [inline tox.ini:2]: Django 3.0 with Python3.5 is an invalid combination

**Pull request merged**